easybuild/tools/modules.py

```diff
@@ -579,19 +579,47 @@ def check_module_output(self, cmd, stdout, stderr):
         """Check output of 'module' command, see if if is potentially invalid."""
         self.log.debug("No checking of module output implemented for %s", self.__class__.__name__)
 
+    def compose_cmd_list(self, args, opts=None):
+        """
+        Compose full module command to run, based on provided arguments
+
+        :param args: list of arguments for module command
+        :return: list of strings representing the full module command to run
+        """
+        if opts is None:
+            opts = []
+
+        cmdlist = [self.cmd, 'python']
+
+        if args[0] in ('available', 'avail', 'list',):
```

Comment:
EasyBuild never uses `av` as subcommand... (see implementation of `available` method)